bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/async/TestAsyncAdmin.java

```diff
@@ -152,8 +152,11 @@ public void testGetTableDescriptor_nonExistingTable() throws Exception {
 
   @Test
   public void testGetTableDescriptor_nullTable() throws Exception {
-    AsyncAdmin asyncAdmin = getAsyncConnection().getAdmin();
-    assertEquals(null, asyncAdmin.getDescriptor(null).get());
+    // This breaks the minicluster, for some reason.
+    if (SharedTestEnvRule.getInstance().isBigtable()) {
```

Comment:
It would be cleaner to use assumptions:
https://github.com/junit-team/junit4/wiki/assumptions-with-assume

```Assume.assumeTrue(SharedTestEnvRule.getInstance().isBigtable())```